pkgs/development/libraries/gnome-sharp/default.nix

```diff
@@ -12,6 +12,8 @@ stdenv.mkDerivation {
 
   patches = [ ./Makefile.in.patch ];
 
+  dontStrip = true;
```

Comment:
Without it, I get there same error as: http://stackoverflow.com/questions/1428535/mono-missing-file-gnome-sharp-at-the-momment-to-compile-the-gnome-sample-appli (as if gnome-sharp wasn't there) when building monodevelop.